distribution/client/test/update.rollback.test.ts

```diff
@@ -0,0 +1,123 @@
+
+import feathers      from '@feathersjs/feathers';
+import mkdirp        from 'mkdirp';
+import tmp           from 'tmp';
+import Update        from '../src/lib/update';
+import HealthChecker from '../src/lib/healthchecker';
+import Storage       from '../src/lib/storage';
+import Status        from '../src/lib/status';
+import Supervisord   from '../src/lib/supervisord';
+
+jest.mock('../src/lib/supervisord');
+
+describe('The update module', () => {
+  let app = null;
+  let update = null;
+
+  beforeEach(() => {
+    const evergreenHome = tmp.dirSync({ unsafeCleanup: true }).name;
+    process.env.EVERGREEN_DATA = evergreenHome;
+    Storage.homeDirectory = (() => evergreenHome);
```

Comment:
Without the `Storage` module being mocked, this seems like a pretty scary override of a module's function. I'm surprised this isn't causing test failures, but I suppose that might be because Jest is running things in parallel processes.

Perhaps it would be safer to `jest.mock` the storage module as well?

I see the same code in `update.test.js`, which means I missed this in a previous set of patches, so it can be addressed in a follow up, but I think it should be addressed.